bundles/pubsub/test/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ celix_bundle_files(pubsub_endpoint_tst
         DESTINATION "META-INF/descriptors"
         )
 celix_bundle_files(pubsub_endpoint_tst
-        meta_data/ping2.properties
+        meta_data/ping3.properties
```

Comment:
I have changed something in the test, to make more readable.
ping2/pong2 ping3/pong3 instead of ping2/pong2 pong2/ping2
To be able to test endpoints (passive) I need this test.
With one publisher and one subscriber I cannot test the full duplex endpoints